e2e/ancestors_test/test_multiple_traverse_ancestors/test.yml

```diff
@@ -0,0 +1,9 @@
+query: >
+    MATCH (c:C)
+    CALL graph_util.ancestors(c) YIELD ancestors
+    UNWIND ancestors as ancestor
+    RETURN ancestor.id AS id
+
+output:
+    - id: 2
+    - id: 1
```

Comment:
Add empty lines here, please. I know tit is hairsplitting, but we try to have these across mage files